lib/SDIR/Translate/translateToSDFG.cpp

```diff
@@ -793,8 +821,6 @@ LogicalResult translation::translateToSDFG(GetAccessOp &op,
   jemit.startNamedObject("attributes");
   if (!(*op).hasAttr("access"))
     jemit.printKVPair("access", "ReadWrite");
```

Comment:
do we need this default?

Reply:
Yes a test fails otherwise. I'll provide all the failing defaults

Reply:
Thank you. We can remove those later.